edge-core/sonic-buildimage
# [Platform][AS9737-32DB]: Revise config.yml and port HW lane list

#### Why I did it
Create_switch() is failed because the config.yml and port HW lane list are not available on community Broadcom SAI.

#### How I did it
1. Refer to the config.yml of AS9736-64D to revise the config.yml of AS9737-32DB.
2. Revise the port HW lane list according to the value get from sairedis.rec.

#### How to verify it
Generate default config for hwsku Accton-AS9737-32DB and Accton-AS9737-32DB-100G, confirm there is no remove port/create port occurred in sairedis.rec

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
